connectors/airwatch/src/test/resources/connector/requests/request.json

```diff
@@ -2,6 +2,6 @@
   "tokens": {
     "udid": ["ABCD"],
     "platform": ["android"],
-    "app_keywords": ["Boxer", "Concur"]
+    "app_keywords": ["Boxer", "Concur", "browser"]
```

Comment:
`browser` app is not configured for `android` platform.
Connector should just continue silently.

Reply:
oh ok, cool, thanks